languages/de.ini

```diff
@@ -1130,6 +1141,7 @@ relais_success_message = "Bestellnummer %%id%% wurde angelegt. Sie erhalten eine
 Related Author = "Ähnliche Verfasser"
 Related Items = "Ähnliche Datensätze"
 Related Subjects = "Ähnliche Schlagworte"
+Relevance = "Relevanz"
```

Comment:
There seem to be a few strings in the German file here which are not present in English. Is this intentional?

Reply:
For those strings the English version is the key itself. So as far as I know they don't need to be translated. E.g. for "Relevance" I would only add Relevance = "Relevance".
Please let me know if it is better to provide a translation in these cases anyway.

Reply:
Please do include translations in these cases as well, even if the key matches the value -- we use the English language file to identify which strings need to be translated into all of the other languages, so if they are omitted from English, they will not be added to any of the other languages during our translation processes.

Reply:
Also consider e.g. the situation where English is a secondary language in a multilingual UI. If a translation is not found for the current language, VuFind will try the fallback languages.

Reply:
Makes sense. Thanks for the explanation :)
I fixed this.